app/__init__.py

```diff
@@ -24,6 +24,36 @@
 ]
 
 
+pic_data = {
```

Comment:
These constant kind of data (`pic_data`, and `about_me`) can live in other file to make the actual code file smaller and cleaner